entitlements.mac.plist

```diff
@@ -4,12 +4,6 @@
 <dict>
     <key>com.apple.security.cs.allow-jit</key>
     <true/>
-    <key>com.apple.security.cs.allow-unsigned-executable-memory</key>
-    <true/>
-    <key>com.apple.security.cs.allow-dyld-environment-variables</key>
-    <true/>
-    <key>com.apple.security.cs.disable-library-validation</key>
```

Comment:
IIRC this was necessary for certain virtual webcam softwares to work.

Reply:
Thanks for the hint, I cross-checked and eg OBS requires it, I'll add it back